tests/test_other.py

```diff
@@ -9631,3 +9631,12 @@ def test_standalone_export_main(self):
     # point.
     # We should consider making this a warning since the `_main` export is redundant.
     self.run_process([EMCC, '-sEXPORTED_FUNCTIONS=[_main]', '-sSTANDALONE_WASM', '-c', path_from_root('tests', 'core', 'test_hello_world.c')])
+
+  def test_dash_o_output(self):
```

Comment:
Lets give this a name that relates to the "-mllvm" problem we are fixing here.

How about "test_llvm_option_dash_o" since this test is specifically about preventing regressions around llvm options that start with `-o`?

Reply:
Done. Thanks for providing guidance :)